src/test/kotlin/io/github/mojira/arisa/LastRunTest.kt

```diff
@@ -0,0 +1,75 @@
+package io.github.mojira.arisa
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.collections.shouldContainExactly
+import io.kotest.matchers.shouldBe
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+class LastRunTest : StringSpec({
+    "should use default time before now if input file is empty" {
+        val lastRun = LastRun(
+            { "" },
+            { }
+        )
+
+        val time = Instant.now().minus(LastRun.DEFAULT_START_TIME_MINUTES_BEFORE_NOW, ChronoUnit.MINUTES)
+
+        lastRun.time.isAfter(time) shouldBe false
```

Comment:
Since this test is already using `LastRun.DEFAULT_START_TIME_MINUTES_BEFORE_NOW`, it could possibly verify that `lastRun.time` is roughly `time` (with a leeway of for example 30 seconds).